demos/tests/args.py

```diff
@@ -68,14 +69,17 @@ def resolve(self, context):
             seq_dir.mkdir(parents=True)
 
             for index, data in enumerate(context.data_sequences[self.sequence_name]):
-                shutil.copyfile(data.resolve(context), str(seq_dir / (name_format % index)))
+                if self.rename:
+                    shutil.copyfile(data.resolve(context), str(seq_dir / (name_format % index)))
+                else:
+                    shutil.copyfile(data.resolve(context), str(seq_dir / (seq[index].stem + seq[0].suffix)))
```

Comment:
I have to keep all original file names, because they are used in ground-truth file.
./ILSVRC2012_val_00000001.JPEG 65
./ILSVRC2012_val_00000002.JPEG 970
./ILSVRC2012_val_00000003.JPEG 230

I don't want to rename each file ILSVRC2012_val_00000001.JPEG ---> input-0001.JPEG.
Tests pass successfully both locally and in CI, so I thought that there are no problems with file names.

Reply:
I understand all this, but it still doesn't make sense for `DataPatternArg` to return a pattern that doesn't match the actual files that it created. I think you should just create a separate `Arg` class that would create a directory with original file names.

Reply:
@IRDonch Please, let me know whether this conversation is resolved or not. I've created a separate class.